acceptance/git/git.go

```diff
@@ -240,6 +241,12 @@ func createGitRepository(ctx context.Context, repositoryName string, files *godo
 			return err
 		}
 
+		// Replace ${GITHOST} with the real git host.
+		// Used in acceptance/examples/default-policy.yaml
+		// Hopefully a noop for all other files.
+		re := regexp.MustCompile(regexp.QuoteMeta("${GITHOST}"))
+		b = []byte(re.ReplaceAllString(string(b), Host(ctx)))
```

Comment:
Could be done using [os.Expand](https://pkg.go.dev/os#Expand) as we do in [other parts](https://github.com/enterprise-contract/ec-cli/blob/fdfc2f2890c7adfac3d17d7b01c001234aa131b4/acceptance/cli/cli.go#L144-L146) of acceptance tests

Reply:
Nice, thanks.

Reply:
Done. (In incoming revision).